.github/CODEOWNERS

```diff
@@ -6,7 +6,7 @@ accounts/scwallet               @gballet
 accounts/abi                    @gballet @MariusVanDerWijden
 cmd/clef                        @holiman
 consensus                       @karalabe
-core/                           @karalabe @holiman @rjl493456442
```

Comment:
Probably unintended?

Reply:
totally intended, just not intended to be merged

Reply:
it's because Martin is a collaborator on my repo and every time I open a PR modifying `core` (which is all the time) against the main branch, Martin is automatically added as a reviewer.